pyensembl/biotypes.py

```diff
@@ -282,6 +282,8 @@
     'known_ncrna',
     # unspliced lncRNAs that are several kb in size.
     'macro_lncRNA',
+    # seems to have been added around Ensembl 81
+    'bidirectional_promoter_lncrna',
```

Comment:
If we search of a Transcript whose biotype isn't in these lists then we get an error. 

Totally open to doing away with this "validation", since Ensembl keeps expanding the lists of valid biotypes.